qfieldsync/qfield_sync.py

```diff
@@ -145,6 +145,11 @@ def __init__(self, iface):
             self.cloud_item_gui_provider
         )
 
+        # autologin
+        dialog = CloudLoginDialog(self.network_manager)
```

Comment:
-1 to auto login. It's intrusive, and potentially a security risk in some environment. 

If we think an auto login is useful for some, let's lock that behavior behind a [x] auto login on QGIS launch setting.

Reply:
I will open another PR with this.

Reply:
see https://github.com/opengisch/qfieldsync/pull/290